ga4gh/datamodel/reads.py

```diff
@@ -187,6 +187,13 @@ def toProtocolElement(self):
         stats.baseCount = None  # TODO requires iterating through all reads
         readGroup.stats = stats
         readGroup.programs = self.getPrograms()
+        experiment = protocol.Experiment()
+        experiment.id = 'TODO'
```

Comment:
Here's my take. In the RG tag we have

CN -> sequencingCentre
DT -> runTime
LB -> library

etc. See the SAM spec on page 3 for this. In the 1000G data we have

LB:2845856850   
DS:SRP001294    
PI:206  
CN:WUGSC
PL:ILLUMINA

We just shove these in as-is. For recordCreateTime just take the file creation time for the parent BAM file.